Add stew

marwanhawari opened the pull request:
> Please check if what you want to add to `awesome-go` list meets [quality standards](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#quality-standards) before sending pull request. Thanks!

**Please provide package links to:**

- repo link (github.com, gitlab.com, etc): https://github.com/marwanhawari/stew
- pkg.go.dev: https://pkg.go.dev/github.com/marwanhawari/stew
- goreportcard.com: https://goreportcard.com/report/github.com/marwanhawari/stew
- coverage service link ([codecov](https://codecov.io/), [coveralls](https://coveralls.io/), [gocover](http://gocover.io/) etc.): https://coveralls.io/github/marwanhawari/stew?branch=main

**Note**: _that new categories can be added only when there are 3 packages or more._

**Make sure that you've checked the boxes below before you submit PR:**
_not every repository (project) will fit into every option, but most projects should_

- [x] I have added my package in alphabetical order.
- [x] I have an appropriate description with correct grammar.
- [x] I know that this package was not listed before.
- [x] I have added pkg.go.dev link to the repo and to my pull request.
- [ ] I have added coverage service link to the repo and to my pull request.
- [x] I have added goreportcard link to the repo and to my pull request.
- [x] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#contribution-guidelines), [maintainers note](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#maintainers) and [Quality standard](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#quality-standards).

Thanks for your PR, you're awesome! :+1:

avelino commented:
Thank you for contributing with [awesome-go](https://github.com/avelino/awesome-go), we will revise your contribution as soon as possible.

Automation body links content check:
- godoc.org or pkg.go.dev: True
- goreportcard.com: True
- coverage: True

> your project is about to be approved, it's under revision, _it may take a few days_

sashamelentyev commented:
review:
repo have't coverage service link

marwanhawari commented:
Switched to coveralls service (from gocover.io) so that I could add a badge to the README.

Updated the original comment above with the coveralls link and the repo also contains a coveralls badge with a link.
